WeakAuras/Prototypes.lua

```diff
@@ -3451,26 +3451,25 @@ Private.event_prototypes = {
         type = "select",
         values = "combatlog_raid_mark_check_type",
         init = "arg",
-        store = true,
```

Comment:
This remove aura_env.state.sourceRaidFlags and will break any existing aura that use it to show raid marker or do something with the info

Reply:
I misunderstood `store` 

I don't think that value should be listed in the tooltip in Display. I thought that's what controlled that.